src/transforms/tag_cardinality_limit/tag_value_set.rs

```diff
@@ -29,8 +63,7 @@ impl AcceptedTagValueSet {
         let storage = match &mode {
             Mode::Exact => TagValueSetStorage::Set(HashSet::with_capacity(value_limit)),
             Mode::Probabilistic(config) => {
-                let num_bits = config.cache_size_per_key / 8; // Convert bytes to bits
```

Comment:
This conversion was wrong and `BloomFilter::with_size` takes the size in bytes, not bits, as an argument.

Reply:
Doh.